third_party/move/tools/move-package/src/compilation/build_plan.rs

```diff
@@ -349,15 +350,18 @@ impl BuildPlan {
 
         // TODO: should inherit color settings from current shell
         let mut error_buffer = Buffer::ansi();
-        if let Err(err) = run_to_yul(&mut error_buffer, MoveToYulOptions {
-            dependencies,
-            named_address_mapping,
-            sources,
-            output: yul_output.clone(),
-            abi_output,
-
-            ..MoveToYulOptions::default()
-        }) {
+        if let Err(err) = run_to_yul(
```

Comment:
Again, seems to be a purely formatting related change. I wonder how this change came to be, if it was not intentional.

Reply:
Emacs formatting doesn't seem to match `rustfmt +nightly`.  I'm fixing it back to make meaningful changes show up better.